.github/workflows/validation-jobs.yml

```diff
@@ -82,8 +82,9 @@ jobs:
           key: ${{ runner.os }}-windows-run-examples-${{ hashFiles('**/Cargo.toml') }}
 
       - name: Build bevy
+        shell: bash
         run: |
-          cargo build --features "bevy_ci_testing"
+          WGPU_BACKEND=dx12 CI_TESTING_CONFIG=./github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing"
```

Comment:
This needs a comment; it's not clear just from reading the code that this *must* be the same as below, or why that's the case.

Reply:
added